### PR TITLE
docs: explain how to start the app with the api key for journey-maps

### PR DESCRIPTION
**Pull request by swiss-chris**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,3 +47,17 @@ Chrome OS: ChromeVox with Chrome.
 ## Testing Supported By
 
 <a href="https://www.browserstack.com/"><img width="160" src="https://user-images.githubusercontent.com/594745/69711802-fc138a80-1101-11ea-9b30-3e90c274737a.png" alt="BrowserStack"/></a>
+
```
Comment (jeripeierSBB) [inline README.md:50]: Thanks for the addition. This kind of information we currently have in DEVELOPER.md where there is already such a section. So I suggest to add your changes there for the moment.

https://github.com/sbb-design-systems/sbb-angular/blob/main/DEVELOPER.md

Comment (swiss-chris) [reply README.md]: Thanks for the input ! I wasn't aware anymore of that DEVELOPER.md file. I moved my text over there.

**Review by jeripeierSBB — APPROVED** [review]
LGTM, thanks

**Comment by swiss-chris** [comment]
@jeripeierSBB Hast du eine Idee wieso diese Tests in diesem PR immer noch fehlschlagen ? Sieht mehr nach einem Datepicker problem aus ...

> //src/angular/datepicker:unit_tests_chromium                             FAILED in 9.5s
>   /home/runner/.cache/bazel/_bazel_runner/ffb965d366b78d9d290bb95018fa337f/execroot/sbb_angular/bazel-out/k8-fastbuild/testlogs/src/angular/datepicker/unit_tests_chromium/test.log
> //src/angular/datepicker:unit_tests_firefox                              FAILED in 17.2s
>   /home/runner/.cache/bazel/_bazel_runner/ffb965d366b78d9d290bb95018fa337f/execroot/sbb_angular/bazel-out/k8-fastbuild/testlogs/src/angular/datepicker/unit_tests_firefox/test.log

**Comment by jeripeierSBB** [comment]
> @jeripeierSBB Hast du eine Idee wieso diese Tests in diesem PR immer noch fehlschlagen ? Sieht mehr nach einem Datepicker problem aus ...
> 
> > //src/angular/datepicker:unit_tests_chromium                             FAILED in 9.5s
> > /home/runner/.cache/bazel/_bazel_runner/ffb965d366b78d9d290bb95018fa337f/execroot/sbb_angular/bazel-out/k8-fastbuild/testlogs/src/angular/datepicker/unit_tests_chromium/test.log
> > //src/angular/datepicker:unit_tests_firefox                              FAILED in 17.2s
> > /home/runner/.cache/bazel/_bazel_runner/ffb965d366b78d9d290bb95018fa337f/execroot/sbb_angular/bazel-out/k8-fastbuild/testlogs/src/angular/datepicker/unit_tests_firefox/test.log

@mhaertwig  Do you have any idea? :/

**Comment by mhaertwig** [comment]
@swiss-chris Sorry, ist im `main` Branch [gefixt](https://github.com/sbb-design-systems/sbb-angular/commit/1959e6940afcd804dc9f5f54dbb4314c98ba055b). War ein Problem in den Tests vom `SbbDatePicker`.

**Pull request merged**